backend/src/executors.py

```diff
@@ -15,13 +15,32 @@
 class ProtBertExecutor(Executor):
     """ProtBERT executor: https://huggingface.co/Rostlab/prot_bert"""
 
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
         log("Initialising ProtBertExecutor.")
-        super().__init__()
+        super().__init__(*args, **kwargs)
+
+        if not os.path.isdir("./models/prot_bert") or not os.path.isdir("./tokenizers/prot_bert"):
+            ProtBertExecutor.download_model()
 
-        self.tokenizer, self.model = self.initialize_executor()
+        self.load_model()
 
-    def initialize_executor(self):
+    def load_model(self):
+        # Assign model from cache
+        self.model = BertModel.from_pretrained("./models/prot_bert")
+
+        # Success logging
+        log("[ProtBertExecutor.load_model] SUCCESS: Loaded model from cache")
+
+        # Assign tokenizer from cache
+        self.tokenizer = BertTokenizer.from_pretrained(
+            "./tokenizers/prot_bert", do_lower_case=False
+        )
+
+        # Success logging
+        log("[ProtBertExecutor.load_model] SUCCESS: Loaded tokenizer from cache")
+
+    @staticmethod
```

Comment:
This method doesn't need to be static. To avoid any problems with Jina, we could avoid making it static.